lib/Raven/Legacy.php

```diff
@@ -14,7 +16,7 @@ class Raven_Client extends \Raven\Client
 {
 }
 
-/**
+/** @noinspection PhpDeprecationInspection
```

Comment:
Thank you for the explanation, now it's much more clear for me. I still don't agree about it, if the inspection warns about a deprecation is for a good reason, in this case both classes are deprecated and should be treated as that. I don't see a good point in having one more line of code just to have a green mark on the IDE, but honestly I don't care too much about it